README.md

```diff
@@ -5,67 +5,35 @@ Exposes high level [OpenStack](http://www.openstack.org/) metrics to [Prometheus
 
 # Deployment
 
-## Requirements
 
-```
-sudo apt-get install python-neutronclient python-novaclient python-keystoneclient python-netaddr
-```
+## Requirements
 
-Install prometheus_client. On Ubuntu 16.04:
-```
-apt-get install python-prometheus-client
-```
+yum install -y gcc
+yum install -y python
+yum install -y python-devel
+yum install -y python-pip
```

Comment:
Instructions for rpm based distros and docker would be great to have. At the same time I think that removal of Ubuntu info is not a good idea.
I think it would be the best to document both approaches so that users can choose whichever method fits their environment best.